conseil-common/src/main/scala/tech/cryptonomic/conseil/common/EvmDomain/Bytecode.scala

```diff
@@ -1,6 +1,6 @@
-package tech.cryptonomic.conseil.common.ethereum.domain
+package tech.cryptonomic.conseil.common.EvmDomain
```

Comment:
why not just `evm` instead of `EvmDomain` ? (btw, as a convention, package name doesn't use camel case)